### ISSUE TITLE
Boot Camp control panel applet is less accessible with NVDA 2019.3.1

### ISSUE THREAD

**Issue by ScottChesworth** [nvaccess/nvda#10798]
### Steps to reproduce:
1. Run NVDA 2019.3.1 on an an Apple computer where Windows 10 has been installed via Boot Camp.
2. Load the Windows control panel.
3. Choose the Boot Camp applet.

### Actual behavior:
1. All tab controls are currently announced as "Unknown".
2. NVDA is no longer interacting as expected with the list box that contains available start up disks (changing selection is silent here).
3. Although the selected start up disk does get read as part of a longer message when I tab to the "Restart" button, speech interrupt doesn't work until NVDA has finished speaking).
4. It doesn't seem to be possible to execute any commands involving the NVDA modifier whilst in the Boot Camp applet. Somehow, the NVDA modifier is being discarded. For example, NVDA+F1 loads Apple's help page in the default browser instead of loading the NVDA log viewer. Keys that wouldn't ordinarily have any function without the NVDA modifier simply do nothing. For example NVDA+S fails to toggle speech mode.

### Expected behavior:
NVDA 2019.2.1 behaviour on all above points would be marvelous, if possible.

#### NVDA installed/portable/running from source:
Portable

#### NVDA version:
2019.3.1

#### Windows version:
Windows 10 (1909)

#### Name and version of other software in use when reproducing the issue:
Apple Boot Camp Services, version 6.0.6136

#### Other information about your system:
Mac Mini 7.1 (from 2014 I think)

#### Does the issue still occur after restarting your PC?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
All previous versions as far back as I can recall worked swimmingly

**Comment by lukaszgo1** [comment]
@ScottChesworth This is caused by the fact that you're using portable version of NVDA> you have to install 2019.3.1 to be able to use this applet.

**Comment by ScottChesworth** [comment]
Gotcha. Thanks @lukaszgo1 . I rely on a couple of add-ons that aren't compatible yet, but that list is getting shorter by the day, so I'll be running a proper install soon enough.
